scheduler/src/cook/mesos/api.clj

```diff
@@ -1916,6 +1986,30 @@
                       :parameters {:query-params JobOrInstanceIds}
                       :handler (destroy-jobs-handler conn is-authorized-fn)}}))
 
+        (c-api/context
```

Comment:
Nitpick: we should consider extracting the `c-api/context` or the `c-api/resource` blocks into separate functions to reduce the size of `main-handler`

Reply:
I agree, but I would favor doing this in a separate PR to keep the diff in this one clean